File: control_flujo.py
'''
    if
    elif
    else
'''

x=2
#debe manejar identado
if x==2:
    print("cumple")

if x==3:
    print("priemra salida")
elif x==2: 
    print("segunda salida")
elif x==8:
    print("tercera salida")
else:
    print("no encontrado")

#OPERADORES LOGICOS
# OR  AND

if (100>10 and 10==100):
    print("doble condicion ok")

if (100>10 or 10==100):
    print("condicion OR ok")

if True:
    print("siempre es verdadero")

#uso de switch case

opcion=8

match opcion:
    case 1:
        print("seleccion 1")
    case 2:
        print("seleccion 2")
    case 3:
        print("seleccion 3")
    case _:
        print("Caso por Defecto")

File: control_flujo.js
/*
    if
    else if
    else
*/

var x=2
//debe estar en llaves
if(x===2)
{
    console.log("cumple")
}

if(x===3)
{
    console.log("primera salida")
}
else if (x==="2")
{
    console.log("primera salida")
}
else if (x===2)
{
    console.log("tercera salida")
}
else
{
    console.log("no contrado")
}


/*OPERADORES LOGICOS
    &&  ||
*/

if (100>10 && 10==100)
{ console.log("doble condicion ok")
}

if (100>10 || 10==100)
{ console.log("condicion or ok")
}

if (true)
{console.log("siempre es verdadero")}

// uso de switch case
let opcion=7;

switch(opcion)
{
    case 1:
        console.log("seleccion 1")
        break
    case 2:
        console.log("seleccion 2")
        break
    case 3:
        console.log("seleccion 3")
        break
    default:
        console.log("Caso por Defecto")
}

